src/how-to-send-file-upload.wiki

```diff
@@ -18,12 +18,12 @@ For example:
 </ocsigen>
 >>
 
-Example:
+Example with Eliom_content.Html5.D module opened:
 <<code language="ocaml"|
 
 let upload = Eliom_service.Http.service
     ~path:["upload"]
-    ~get_params:unit
+    ~get_params:Eliom_parameter.unit
```

Comment:
That is fine with respect to the old `Eliom_service` API. Good for now. In the future, you can do PRs against the `stable` branch if you want to improve the docs for Eliom 5.0 or contribute to the docs for 6.0 by using the new API :).